gateway/ln-gateway/src/state_machine/events.rs

```diff
@@ -0,0 +1,46 @@
+use fedimint_core::core::ModuleKind;
+use fedimint_core::Amount;
+use fedimint_eventlog::{Event, EventKind};
+use fedimint_ln_common::contracts::outgoing::OutgoingContractAccount;
+use fedimint_ln_common::contracts::ContractId;
+use serde::{Deserialize, Serialize};
+
+use super::pay::OutgoingPaymentError;
+
+#[derive(Serialize, Deserialize)]
+pub struct OutgoingPaymentStarted {
+    pub contract_id: ContractId,
+    pub invoice_amount: Amount,
+}
```

Comment:
Should contain operation id so e.g. invoice can be looked up from operation log.

Reply:
Fixed!

Reply:
You only added the operation id to the start event, which is ok I guess as long as consumers keep an index `payment_hash -> operation_id`. Worth considering the storage/ease of consumption trade-off though, cc @dpc.